Apps/CesiumViewer/CesiumViewer.js

```diff
@@ -181,15 +181,15 @@ define([
         var camera = viewer.camera;
```

Comment:
Rather than pass camera everywhere, just move this declaration to be above `saveCamera`, it will always be the same camera object.